content/200-concepts/100-components/250-preview-features/050-client-preview-features.mdx

```diff
@@ -44,4 +43,107 @@ To enable a Prisma Client Preview feature:
    npx prisma generate
    ```
 
-3. If you are using Visual Studio Code and the Preview feature is not available in your `.ts` file after generating the client, run the **TypeScript: Restart TS server** command.
+3. If you are using Visual Studio Code and the preview feature is not available in your `.ts` file after generating the client, run the **TypeScript: Restart TS server** command.
+
+## Preview features promoted to general availability
+
+In the list below, you can find a history of Prisma Client features that were in preview and are now in general availability.
```

Comment:
Totally OT and probably future facing:

The list could become a table:

```
preview feature | preview | general availability | comments
```

Most would only list the name, and preview and GA version - and some then have a list in the "comments" (or wahtever other name we could give it)